docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -198,33 +249,11 @@ Exit out of CQLSH:
 
 `cqlsh:medusa_test> exit`
 
-Review the current charts that are in use, so far:
-
-`helm list`
-
-```
-NAME               	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                  	APP VERSION
-k8ssandra          	default  	1       	2021-02-03 04:17:23.107265 -0700 MST	deployed	k8ssandra-0.38.0        3.11.7  
-```
-
-Also get the deployment status, so far:
-
-`kubectl get deployment`
-```
-NAME                                       READY   UP-TO-DATE   AVAILABLE   AGE
-cass-operator                              1/1     1            1           159m
-k8ssandra-grafana-operator-k8ssandra       1/1     1            1           159m
-k8ssandra-kube-prometheus-stack-operator   1/1     1            1           159m
-k8ssandra-reaper-k8ssandra                 1/1     1            1           156m
-k8ssandra-reaper-operator-k8ssandra        1/1     1            1           159m
-grafana-deployment                         1/1     1            1           158m
-```
-
-The output above shows the addition of medusa-test-medusa-operator-k8ssandra pod. 
+**TIP:** Keep the sample `medusa_test.users` table data in mind -- we will use subsequent backup and restore steps with this data.
 
 ### Create the backup
 
-Now create a backup using a `test` chart:
+Now create a backup by referencing the backup chart in GitHub:
```

Comment:
We should be referencing the chart repo in the docs. The command below should be:

```
helm install test k8ssandra/backup --set name=test,cassandraDatacenter.name=dc1
```

Reply:
Inc'd correct command - thanks John